tools/crm_shadow.c

```diff
@@ -237,6 +237,33 @@ query_real_cib(xmlNode **output, GError **error)
     return rc;
 }
 
+/*!
+ * \internal
+ * \brief Write the shadow XML to a file
+ *
+ * \param[in,out] xml       Shadow XML
+ * \param[in]     filename  Name of destination file
+ * \param[in]     reset     Whether the write is a reset (for logging only)
+ * \param[out]    error     Where to store error
+ */
+static int
+write_shadow_file(xmlNode *xml, const char *filename, bool reset,
+                  GError **error)
+{
+    int rc = write_xml_file(xml, filename, FALSE);
+
+    rc = pcmk_legacy2rc(rc);
+    if (rc < 0) {
```

Comment:
Nope, I already made that mistake :) But I do need to move the `pcmk_legacy2rc()` inside the if block.

`write_xml_file()` returns the number of bytes written on success, -errno otherwise.